### PR TITLE
Hard Mode Rule Adjustment

### PR DESCRIPTION
**Pull request by Skinnern**
allow users to move all cards to other columns on the first draw only

### CODE REVIEW

**Comment by Skinnern** [comment]
just noticed while fixing up some spacing that an error message incorrectly displays when you move a card on the first turn, will fix shortly

**Comment by Skinnern** [comment]
Fixed the error display issue, and believe it is good to go now

File: src/main/java/models/HardGame.java [hunk 1]
```diff
@@ -4,17 +4,30 @@
     public void move(int columnFrom, int columnTo) {
         // check if the to and from values are in range
         // we'll have to subract one from these values to access the column at the correct index
-        if (columnFrom >= 0 && columnFrom <= 3 && columnTo >= 0 && columnTo <= 3) {
-            if (!super.columnHasCards(columnTo)) {
-                if (super.getTopCard(columnFrom).value == 14){
-                    System.out.println("Moving card");
+
+        //if first draw
```
Comment (danvanhorn) [inline src/main/java/models/HardGame.java]: This definitely works! Good job man.

File: src/main/java/models/HardGame.java [hunk 2]
```diff
@@ -4,17 +4,30 @@
...
+        if (super.deck.size() == 48) {
+            if (columnFrom >= 0 && columnFrom <= 3 && columnTo >= 0 && columnTo <= 3) {
```
Comment (danvanhorn) [inline src/main/java/models/HardGame.java]: ^^^ This definitely works! Good job man. ^^^

**Pull request merged**